mmeval/metrics/instance_seg.py

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+from copy import deepcopy
+from typing import Dict, List, Sequence
+
+from mmeval.core.base_metric import BaseMetric
+from mmeval.metrics._vendor.scannet import scannet_eval
+
+
+def aggregate_predictions(masks, labels, scores, valid_class_ids):
+    """Maps predictions to ScanNet evaluator format.
+
+    Args:
+        masks (list[numpy.ndarray]): Per scene predicted instance masks.
+        labels (list[numpy.ndarray]): Per scene predicted instance labels.
+        scores (list[numpy.ndarray]): Per scene predicted instance scores.
+        valid_class_ids (tuple[int]): Ids of valid categories.
+
+    Returns:
+        list[dict]: Per scene aggregated predictions.
+    """
```

Comment:
It is recommended to write this function as a class method～

Reply:
fixed